### PR TITLE
Add eu.quelltext.open_chakra_toning

### PR DESCRIPTION
**Pull request by niccokunzmann**
### Please confirm your submission meets all the criteria

This app plays different sounds and guides people to meditate to open their chakras.

- [x] Please describe your application briefly.
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

Hello, this is my first app submission to flathub. Thanks for having a look and maintaining this! If there is anything in the way of submitting this, please let me know!

### CODE REVIEW

File: eu.quelltext.open_chakra_toning.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  # Wayland access
+  - --socket=wayland
+  # GPU acceleration if needed
+  - --device=dri
+  # Needs to talk to the network:
+#  - --share=network
+  # Needs to save files locally
+#  - --filesystem=xdg-documents
+  # Needs audio
+  - --socket=pulseaudio
```
Comment (bbhtt) [inline eu.quelltext.open_chakra_toning.yml:43]: This should be on top, before modules.

File: eu.quelltext.open_chakra_toning.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+sdk: org.freedesktop.Sdk
+command: open_chakra_toning
+modules:
+  - name: open_chakra_toning
+    buildsystem: simple
+    build-commands:
+      - ./inside.sh
+    sources:
+      - type: archive
+        url: https://github.com/niccokunzmann/open_chakra_toning/releases/download/v1.0.0/Open-Chakra-Toning-linux-x64.zip
```
Comment (bbhtt) [inline eu.quelltext.open_chakra_toning.yml:28]: What happened to aarch64?

File: eu.quelltext.open_chakra_toning.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+id: eu.quelltext.open_chakra_toning
```
Comment (bbhtt) [inline eu.quelltext.open_chakra_toning.yml:16]: How do we verify quelltext.eu belongs to the project? It redirects to https://hosted.quelltext.eu/login, maybe put some thing up?

**Comment by niccokunzmann** [comment]
This is my domain. All my apps use that so I do not conflict. Everything in time!

**Comment by niccokunzmann** [comment]
Thanks for reviewing! Is this a should or a must?

I just thought about the metainfo rewriter that F-Droid uses to standardize the yml files. Do you have that or an issue for it?

Please tell me clearly what must [RFC style] be changed and what is subjective or nice. That will help me a great deal with clarity!

- position of finish-args
- aarch64
- domain

Is there documentation on guidelines for that? I like to document stuff!

**Comment by hfiguiere** [comment]
> This is my domain. All my apps use that so I do not conflict. Everything in time!

Now is the time.

**Comment by hfiguiere** [comment]
> - position of finish-args 

As asked on the review it should be above `modules`.

> - aarch64 

Build from source.

> - domain Is there documentation on guidelines for that? I like to document stuff!

You acknowledged the section about that it has all the links to it. If we can verify easily because it's parked or whatever, we'll ask about it.

**Comment by niccokunzmann** [comment]
Thanks for your support! I will take some time to get on with the feelings around the submission and will come back to you very likely after a while!

## ID

I do not understand: 

>  the app author has control over **or** where their app is hosted. - https://docs.flathub.org/docs/for-app-authors/requirements/#application-id

I see no need to change the id according to the documentation. Could you help me?

## aarch64

Ok, I hear that aarch64 is required for submission. This will take a while longer. 
I also opened an issue because this is a flutter app and I would like to be able to look at the example to tell me how to do that. https://github.com/Merrit/flutter_flatpak_example/issues/10

## position of finish-args

Will be done!

File: eu.quelltext.open_chakra_toning.yml (hunk 3 above)
Comment (niccokunzmann) [reply eu.quelltext.open_chakra_toning.yml:16]:
```
$ ping -4 quelltext.eu
PING  (78.47.87.181) 56(84) bytes of data.
```

image

Comment (niccokunzmann) [reply eu.quelltext.open_chakra_toning.yml:16]: I can also move the hosting of the Github pages over.
I did not know that domain verification is required.

Comment (bbhtt) [reply eu.quelltext.open_chakra_toning.yml:16]: I don't know what the screenshot is supposed to show. We need to check that you or the project owns the domain used and it's not someone else's. That's what is meant by the app author controls the domain.

The way to do that would be to link the repo or make a page about the app on the domain.

Pinging doesn't prove anything, I can ping example.org, doesn't mean I own the domain.

Comment (hfiguiere) [reply eu.quelltext.open_chakra_toning.yml:16]: >  I did not know that domain verification is required.

You acknowledged:

> The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

But without a way to simply verify that... 

People put a webpage for the software, some link to what they do, etc. So it's often easy to reconcile the author with the domain. If it is parked mostly it's impossible to verify.

Comment (niccokunzmann) [reply eu.quelltext.open_chakra_toning.yml:16]: Yeah, I see that. Thanks... I did not know that. E.g. when I submitted to F-Droid, they were file with the unique id I used for my apps. We can just wait and I will host it there, then :) Thanks for taking the time!

Comment (niccokunzmann) [reply eu.quelltext.open_chakra_toning.yml:16]: http://open-chakra-toning.quelltext.eu/ shows the app now. I think, this should resolve the issue.

**Comment by niccokunzmann** [comment]
I think that I would like to add the arm build later. If you wish, you can wait for that release. Otherwise, I will continue and submit another yml file.

**Comment by bbhtt** [comment]
bot, build eu.quelltext.open_chakra_toning

**Pull request closed without merging**